Meadowlark-js/backends/meadowlark-local-mongodb-opensearch/src/repository/Db.ts

```diff
@@ -5,30 +5,30 @@
 
 import { Collection, Db, MongoClient } from 'mongodb';
 import { Logger } from '@edfi//meadowlark-core';
-import { Document } from '../model/Document';
+import { MeadowlarkDocument } from '../model/MeadowlarkDocument';
 
 const MONGO_URL: string = process.env.MONGO_URL ?? 'mongodb://mongo1:27017,mongo2:27018,mongo3:27019/?replicaSet=rs0';
 
-const client = new MongoClient(MONGO_URL);
+const client = new MongoClient(MONGO_URL, { w: 'majority', readConcern: 'majority' });
```

Comment:
Should we make this consistency setting configurable?

Reply:
Not sure. Majority is required for transaction support. Not sure which others qualify, but not all do.